<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditorMainClass</class>
 <widget class="QMainWindow" name="EditorMainClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>425</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EditorMain</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="MainMenuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="MainMenu">
    <property name="title">
     <string>文件</string>
    </property>
    <widget class="QMenu" name="Menu_CreateNew">
     <property name="title">
      <string>新建</string>
     </property>
     <addaction name="CreateNewLevel"/>
     <addaction name="CreateNewWorld"/>
    </widget>
    <addaction name="Menu_CreateNew"/>
    <addaction name="OpenWorld"/>
    <addaction name="SaveWorld"/>
    <addaction name="SaveAsWorld"/>
    <addaction name="OpenProgramDir"/>
    <addaction name="CompileAllShader"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>编辑</string>
    </property>
    <addaction name="ResetWindowStyle"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>工具</string>
    </property>
    <addaction name="MemoryWatcher"/>
   </widget>
   <addaction name="MainMenu"/>
   <addaction name="menu_3"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="CreateNewWorld">
   <property name="text">
    <string>新建世界</string>
   </property>
  </action>
  <action name="SaveWorld">
   <property name="text">
    <string>保存世界</string>
   </property>
  </action>
  <action name="SaveAsWorld">
   <property name="text">
    <string>世界另存为</string>
   </property>
  </action>
  <action name="ResetWindowStyle">
   <property name="text">
    <string>重新载入样式表</string>
   </property>
  </action>
  <action name="OpenProgramDir">
   <property name="text">
    <string>打开程序路径</string>
   </property>
  </action>
  <action name="CompileAllShader">
   <property name="text">
    <string>重新编译所有着色器</string>
   </property>
  </action>
  <action name="CreateNewLevel">
   <property name="text">
    <string>新建关卡</string>
   </property>
  </action>
  <action name="OpenWorld">
   <property name="text">
    <string>打开世界</string>
   </property>
  </action>
  <action name="MemoryWatcher">
   <property name="text">
    <string>内存观察器</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="EditorMain.qrc"/>
 </resources>
 <connections/>
</ui>
